WordPressAuthenticator/NUX/NUXButton.swift

```diff
@@ -1,54 +1,285 @@
 import UIKit
 import WordPressShared
+import WordPressUI
+
+/// A protocol for an element that can display a UIActivityIndicatorView
+@objc public protocol ActivityIndicatorButton {
```

Comment:
@mindgraffiti since the protocol `ActivityIndicatorButton` isn't really used anywhere (there are no ivars / vars anywhere with that specific Type), can we nuke it?

Reply:
✔️